pywinauto/controls/HwndWrapper.py

```diff
@@ -1213,6 +1213,15 @@ def set_focus(self):
 
         return self
 
+    def has_keyboard_focus(self):
+        if self.get_focus() == self.handle:
```

Comment:
`get_focus` is not a wrapper around Win32 API function `GetFocus`. It's a completely different thing.